clients/drcachesim/tracer/instru_offline.cpp

```diff
@@ -955,8 +955,7 @@ offline_instru_t::identify_elidable_addresses(void *drcontext, instrlist_t *ilis
         // view by expanding the instr in raw2trace (e.g. using
         // drx_expand_scatter_gather) when building the ilist.
         if (drutil_instr_is_stringop_loop(instr)
-            // TODO i#5036: Scatter/gather support incomplete on AArch64.
-            IF_X86(|| instr_is_scatter(instr) || instr_is_gather(instr))) {
+                IF_X86_OR_AARCH64(|| instr_is_scatter(instr) || instr_is_gather(instr))) {
```

Comment:
All archs define instr_is_scatter and instr_is_gather, and return default false if not implemented. How about we just remove the conditional compilation here and below?

Reply:
The riscv64 implementations have `ASSERT_NOT_IMPLEMENTED(false)` in so wouldn't taking out the conditional compilation stop drcachesim working on riscv64?

Reply:
Hmm, right. Okay, let's keep it the way it is.